components/installer/pkg/config/v1/load.go

```diff
@@ -50,6 +50,9 @@ func LoadMock() *Config {
 			Kind: ObjectRefSecret,
 			Name: "https-certs",
 		},
+		OpenVSX: OpenVSX{
+			URL: "openvsx-testing.com",
```

Comment:
Why not `https://open-vsx.org`?

Reply:
This is just mock configuration and not used anywhere, right? 🤔 I followed the pattern with other fields!

Reply:
Indeed, it is just a mock. Other than the domain, the remaining values "make sense". 
The original comment is a nit, really :)

Reply:
Updated! :)